### PR TITLE
Update links, and gitignore

### PR DESCRIPTION
**Pull request by emceeaich**
#### Details
This PR adds/updates:
- The docs/doctree path to .gitignore since these are generated files
- Project links to point at correct bmo products/components and project links

#### Additional info
N/A

#### Questions
- What should be the value of `repository.tests` in `./contribute.json`
- What should be in `urls` in `./contribute.json`

#### Test Plan
Run `./makedocs.pl` from ./docs path in repository

### CODE REVIEW

**Comment by justdave** [comment]
>     * What should be the value of `repository.tests` in `./contribute.json`

Probably `https://github.com/bugzilla/bugzilla/actions` or `https://github.com/bugzilla/harmony/actions`
There's also `https://app.circleci.com/pipelines/github/bugzilla/bugzilla` and `https://app.circleci.com/pipelines/github/bugzilla/harmony` ...

>     * What should be in `urls` in `./contribute.json`

Uhh, remove them? We don't really have an equivalent.

There is a contribute.json at https://www.bugzilla.org/contribute.json
We should probably merge some of the values.

**Comment by emceeaich** [comment]
I've updated `repository.tests` and removed `urls`. Will review a merge with the `contribute.json` in the docs site in a separate PR.

File: contribute.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+    "description": "The software solution designed to drive software development, which lets you plan, organize and release software on your own teams' schedule.",
     "repository": {
-        "url": "https://github.com/mozilla-bteam/bmo",
+        "url": "https://github.com/bugzilla/harmony",
         "license": "MPL2",
         "type": "git",
-        "clone": "https://github.com/mozilla-bteam/bmo.git",
-        "tests": "https://treeherder.mozilla.org/#/jobs?repo=bmo-master"
+        "clone": "https://github.com/bugzilla/harmony.git",
+        "tests": "https://app.circleci.com/pipelines/github/bugzilla/harmony"
```
Comment (justdave) [inline contribute.json:9]: Hate to do this to you now, but I just committed a change that yanked all the circleci tests in favor of doing it all in GitHub Actions some GHA can do everything we're doing in the tests.

So make this https://github.com/bugzilla/harmony/actions

**Review by justdave — CHANGES_REQUESTED** [review]

File: contribute.json (hunk 1 above)
Comment (emceeaich) [reply contribute.json]: Done.

**Review by justdave — APPROVED** [review]

**Pull request merged**